www/master/components/dashboard/views/replication.html

```diff
@@ -49,9 +58,13 @@
           <tr>
             <td class="name">Related Pods</td>
             <td class="value">
-              <div ng-repeat="(label, value) in replicationController.desiredState.replicaSelector">
-                  <a ng-href="#/dashboard/groups/type/selector/{{label}}={{value}},type=pod">{{label}}: {{value}}</a>
-                </div>
+              <div ng-show="replicationController.spec.selector && Object.keys(replicationController.spec.selector).length > 1">
+                <a ng-href="#/dashboard/groups/type/selector/{{getSelectorUrlFragment(replicationController.spec.selector)}},type=pod">{{getSelectorUrlFragment(replicationController.spec.selector)}}</a>
+              </div>
+
+              <div ng-repeat="(label, value) in replicationController.spec.selector">
+                <a ng-href="#/dashboard/groups/type/selector/{{label}}={{value}},type=pod">{{label}}={{value}}</a>
```

Comment:
Looks like you say here "label=value" where above (in line 53) you say "label: value"

It would be good to be consistent here.